app/views/workflows/_edit_form.erb

```diff
@@ -4,7 +4,7 @@
     <%= f.alert_message "Please fix the errors below." %>
 
     <%= f.text_field :name %>
-    <%= f.select :batch_host, OODClusters.map { |key, val| [ "#{val.title} (#{val.resource_mgr_server.host})", key ] }, { selected: @workflow.batch_host, label: "Batch Server" }, { class: "selectpicker" } %>
+    <%= f.select :batch_host, OODClusters.map { |cluster| [ "#{cluster.metadata_config[:title]} (#{cluster.job_config[:host]})", cluster.id ] }, { selected: @workflow.batch_host, label: "Batch Server" }, { class: "selectpicker" } %>
```

Comment:
`job_config` should never be used here, as the actual config options will vary depending on the adapter

I see before we showed "Oakley (oak-batch.osc.edu)" and now we would possibly just be showing "Oakley" but that is probably okay.

Also, the label of this field should probably change from "Batch Server" to "Cluster" so it matches where we use it in other places. (and change Job Options label to say Cluster as well)